tests/unit/meta/Dimensions.ts

```diff
@@ -2,13 +2,34 @@ import global from '@dojo/shim/global';
 import * as registerSuite from 'intern!object';
 import * as assert from 'intern/chai!assert';
 import { stub } from 'sinon';
-import { v } from '../../../src/d';
-import { ProjectorMixin } from '../../../src/main';
 import Dimensions from '../../../src/meta/Dimensions';
-import { WidgetBase } from '../../../src/WidgetBase';
-import { ThemeableMixin } from './../../../src/mixins/Themeable';
+import NodeHandler, { Type } from '../../../src/NodeHandler';
 
 let rAF: any;
+const defaultDimensions = {
```

Comment:
Don't we export these from dimensions?

Reply:
no, i did add that earlier but removed again, makes sense